Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DataBaseService.DbModels;
+using DataBaseService.Interfaces;
+using DTO;
+
+namespace DataBaseService.Repositories
+{
+    public class UserRepository : IRepository<UserEmailPassword>
+    {
+        readonly IMapper<UserEmailPassword, DbUser> mapper;
+        readonly DataBaseContext dbContext;        
+        
+        public UserRepository(IMapper<UserEmailPassword, DbUser> mapper, DataBaseContext dbContext)
+        {
+            this.mapper = mapper;
+            this.dbContext = dbContext;
+        }
+        public void Create(UserEmailPassword data)
```

Comment:
nullcheck? Exceptions?

Reply:
UserAlreadyExist??

Reply:
Change table: Id and Email must be unique
Add try catch for create in UserCredentialRepository